pandas/tests/series/test_analytics.py

```diff
@@ -1330,6 +1330,13 @@ def test_duplicate_keep_all_ties(self):
         expected = Series([6, 7, 7, 7, 7], index=[7, 3, 4, 5, 6])
         assert_series_equal(result, expected)
 
+    def test_boolean(self):
+        # GH 26154 : ensure True > False
```

Comment:
can you add some more elements here [True, False, True, True] as another case

Reply:
Update the test with `pytest.mark.parametrize`